lib/msal-common/src/request/RequestValidator.ts

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Considering moving these methods to RequestUtils. Thoughts?

Reply:
IMO this file here is actually more representative of a `RequestUtils` file and the other (current) `RequestUtils` file is really just a bunch of helper/setter functions that should probably live where that request it being built

Reply:
I've refactored away from using the static methods and just created a "ParametersBuilder" that contains these methods, as they will be shared by all the clients in msal-common